.travis.yml

```
@@ -1,6 +1,6 @@
 language: python
 python:
-  - "3.5"
+  - "3.6"
```

Could we add the 3.7 in the list? 
```suggestion
  - "3.6"
  - "3.7"
```